bindings/wasm/src/credential/credential.rs

```diff
@@ -18,7 +18,7 @@ use crate::error::Result;
 use crate::error::WasmResult;
 
 #[wasm_bindgen(js_name = Credential, inspectable)]
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct WasmCredential(pub(crate) Credential);
```

Comment:
Why does this derive `Serialize` and `Deserialize` (and `PartialEq` for that matter)? We always (de)serialize the inner rust type, not the Wasm wrapper.